problem-29/README.md

```diff
@@ -0,0 +1,15 @@
+# Task 29: Dynamically add class depending on if mouse is hovering element (medior 1)
+
+## Description
+Update the BackgroundViewComponent HTML component only. Add Angular events so that when a mouse enters or leaves the inner container, a function is fired.
```

Comment:
After reading this, it is very difficult to understand what needs to be done.

Reply:
Resolved to make read me more clear - let me know if question is to complicated and needs to be replaced